subsys/bluetooth/controller/hci_internal.c

```diff
@@ -772,7 +772,10 @@ static void cmd_put(uint8_t *cmd_in, uint8_t * const raw_event_out)
 {
 	uint8_t status;
 	uint16_t opcode = sys_get_le16(cmd_in);
-	uint8_t return_param_length = CMD_COMPLETE_MIN_SIZE;
+
+	/* Assume command complete */
+	uint8_t return_param_length = sizeof(struct bt_hci_evt_cmd_complete)
+				      + sizeof(uint8_t);
```

Comment:
Could use the `bt_hci_evt_cc_status` to make it clear that the uint8_t is the status, since status is not part of the command complete struct.